test

```diff
@@ -1 +1,11 @@
+if [[ $WITH_COVER_OPTIONS ]]; then
+    cover_deps=()
+    perl -MDevel::Cover -e1 &> /dev/null || cover_deps+=('perl(Devel::Cover)')
+    perl -MDevel::Cover::Report::Codecovbash -e1 &> /dev/null || cover_deps+=('perl(Devel::Cover::Report::Codecovbash)')
+    [[ $cover_deps ]] && zypper --non-interactive in "${cover_deps[@]}"
```

Comment:
Wouldn't it be enough to just do the zypper install, and it won't do anything if the modules are installed?

Reply:
I want to use this script also locally and then it is really inconvenient having to wait for zypper to refresh repos and run the resolver to find out that certain packages are already installed. Maybe there should be a different script for the CI and for local runs but for now I've just stuck with this approach.